vectordraw.py

```diff
@@ -2,6 +2,28 @@
 # script tag of the problem XML definition - see api-example.xml for
 # an example.
 
+import json
+import math
+
+
+try:
+    custom_checks
```

Comment:
How could this possibly be defined at this point?  Can't we just have `custom_checks = {}` here with a comment that this can be customised?  Same thing applies to `success_message`.